sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -141,9 +142,14 @@ def _cloud_event_to_generated(cloud_event, **kwargs):
         **kwargs
     )
 
-def _from_cncf_events(event):
+def _from_cncf_events(event): # pylint: disable=inconsistent-return-statements
     """This takes in a CNCF cloudevent and returns a dictionary.
     If cloud events library is not installed, the event is returned back.
+
+    :param event: The event to be serialized
+    :type event: cloudevents.CloudEvent
```

Comment:
Not sure if this is referring to the CloudEvents in core or if there is one defined in EG (and I don't see a `cloudevents` imported in this file) but I believe it should look more like this:
```suggestion
    :type event: ~azure.core.messaging.CloudEvent
```

Reply:
This is referring to a CNCF cloudEvent from the package cloudevents but it should be cloudevents.http.CloudEvent